charts/stable/firefly-iii/Chart.yaml

```
@@ -0,0 +1,43 @@
+---
+apiVersion: v2
+appVersion: 5.6.14
```

```suggestion
appVersion: version-5.6.14
```
We recommend putting the image tag as the appVersion and leaving the tag blank in the values, as it inherits this line.